_observing-your-data/alerting/triggers.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: default
+title: Triggers
+nav_order: 10
+grand_parent: Alerting
+parent: Monitors
+redirect_from:
+  - /monitoring-plugins/alerting/monitors/
+---
+
+# Triggers
+
+How you create a trigger differs depending on the monitor method selected when the monitor was created. The monitor methods are **Visual editor**, **Extraction query editor**, and **Anomaly detector**. Learn more about each type in the following sections.
+
+## Creating triggers
+
+To create a trigger:
+
+1. In the **Create monitor** window, select **Add trigger**.
+2. Enter the trigger name, severity level, and trigger condition. Severity levels help manage alerts. For example, a trigger with a high severity level (for example, 1) may notify a specific individual, whereas a trigger with a low severity level might notify a chat room.
```

Comment:
Is there a range for severity levels? If it's a dropdown list, then there's no need to specify it here but if it's a field, we might want to include the range here.

Reply:
@AWSHurneyt Please advise on this question.

Reply:
Severity ranges from 1 (Highest) to 5 (Lowest)